codex-ui/src/vue/components/row/Row.vue

```diff
@@ -102,6 +102,7 @@ defineProps<{
   }
 
   &__center {
+    min-height: 20px;
```

Comment:
don't hardcode it inside a Row component. Row's slot can contain any element, so preventing jumping should be placed in a component where slot is used.